Source/WebCore/rendering/svg/SVGRenderSupport.cpp

```diff
@@ -55,16 +55,16 @@
 
 namespace WebCore {
 
-LayoutRect SVGRenderSupport::clippedOverflowRectForRepaint(const RenderElement& renderer, const RenderLayerModelObject* repaintContainer)
+LayoutRect SVGRenderSupport::clippedOverflowRectForRepaint(const RenderElement& renderer, const RenderLayerModelObject* repaintContainer, RenderObject::VisibleRectContext context)
 {
     // Return early for any cases where we don't actually paint
     if (renderer.style().visibility() != Visibility::Visible && !renderer.enclosingLayer()->hasVisibleContent())
         return LayoutRect();
 
     // Pass our local paint rect to computeFloatVisibleRectInContainer() which will
     // map to parent coords and recurse up the parent chain.
-    FloatRect repaintRect = renderer.repaintRectInLocalCoordinates();
-    return enclosingLayoutRect(renderer.computeFloatRectForRepaint(repaintRect, repaintContainer));
+    auto repaintRect = renderer.repaintRectInLocalCoordinates(context.options.contains(RenderObject::VisibleRectContextOption::CalculateFastRepaintRect) ? RenderObject::RepaintRectCalculation::Fast : RenderObject::RepaintRectCalculation::Accurate);
```

Comment:
I think you can add an inline method in `VisibleRectContext`  to return `RepaintRectCalculation`. So this code can look like this:
```
auto repaintRect = renderer.repaintRectInLocalCoordinates(context.repaintRectCalculation());
```

Reply:
Changed.